Makefile

```diff
@@ -374,6 +379,7 @@ ADDONS_NAPI_BINDING_SOURCES := \
 	$(filter-out test/addons-napi/??_*/*.cc, $(wildcard test/addons-napi/*/*.cc)) \
 	$(filter-out test/addons-napi/??_*/*.h, $(wildcard test/addons-napi/*/*.h))
 
+.PHONY: test/addons-napi/.buildstamp
```

Comment:
If we're going to make this target `.PHONY`, I would remove the `touch` so it's no longer a file and probably rename the target so it doesn't look like a file either.